### ISSUE TITLE
1.4.0 unable to open database

### ISSUE THREAD

**Issue by coreform** [facebook/stetho#433]
Following an update from

```
compile 'com.facebook.stetho:stetho:1.3.1'
compile 'com.facebook.stetho:stetho-okhttp3:1.3.1'
```

to 

```
compile 'com.facebook.stetho:stetho:1.4.0'
compile 'com.facebook.stetho:stetho-okhttp3:1.4.0'
```

I get the following crash when attempting to open any SQLiteDatabase in an Android app, Android 6.0.1:

```
09-13 13:55:47.289 7425-9609/com.just.another.app D/ChromeDevtoolsServer: onOpen
09-13 13:55:47.574 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Page.canEmulate
09-13 13:55:47.592 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Console.setTracingBasedTimeline
09-13 13:55:47.612 7425-9609/com.just.another.app I/ResponseBodyFileManager: Cleaned up temporary network files.
09-13 13:55:47.690 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Debugger.setAsyncCallStackDepth
09-13 13:55:47.702 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Debugger.skipStackFrames
09-13 13:55:47.714 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Runtime.enable
09-13 13:55:47.756 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Worker.enable
09-13 13:55:47.759 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Timeline.enable
09-13 13:55:47.786 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.enable
09-13 13:55:47.787 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Page.getNavigationHistory
09-13 13:55:47.789 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Worker.setAutoconnectToWorkers
09-13 13:55:47.791 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.792 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.804 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.805 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.806 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.808 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.809 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.810 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: IndexedDB.requestDatabaseNames
09-13 13:55:47.811 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: Page.getNavigationHistory
09-13 13:55:47.812 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: ApplicationCache.enable
09-13 13:55:47.814 7425-9609/com.just.another.app D/ChromeDevtoolsServer: Method not implemented: Not implemented: ApplicationCache.getFramesWithManifests
09-13 13:55:51.760 7425-9609/com.just.another.app E/SQLiteLog: (14) cannot open file at line 31278 of [2ef4f3a5b1]
09-13 13:55:51.760 7425-9609/com.just.another.app E/SQLiteLog: (14) os_unix.c:31278: (2) open(/challenges.db) - 
09-13 13:55:51.767 7425-9609/com.just.another.app E/SQLiteDatabase: Failed to open database '/challenges.db'.
                                                                                         android.database.sqlite.SQLiteCantOpenDatabaseException: unknown error (code 14): Could not open database
                                                                                             at android.database.sqlite.SQLiteConnection.nativeOpen(Native Method)
                                                                                             at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:213)
                                                                                             at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:197)
                                                                                             at android.database.sqlite.SQLiteConnectionPool.openConnectionLocked(SQLiteConnectionPool.java:463)
                                                                                             at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:185)
                                                                                             at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:177)
                                                                                             at android.database.sqlite.SQLiteDatabase.openInner(SQLiteDatabase.java:806)
                                                                                             at android.database.sqlite.SQLiteDatabase.open(SQLiteDatabase.java:791)
                                                                                             at android.database.sqlite.SQLiteDatabase.openDatabase(SQLiteDatabase.java:694)
                                                                                             at android.database.sqlite.SQLiteDatabase.openDatabase(SQLiteDatabase.java:669)
                                                                                             at com.facebook.stetho.inspector.database.DefaultDatabaseConnectionProvider.performOpen(DefaultDatabaseConnectionProvider.java:64)
                                                                                             at com.facebook.stetho.inspector.database.DefaultDatabaseConnectionProvider.openDatabase(DefaultDatabaseConnectionProvider.java:32)
                                                                                             at com.facebook.stetho.inspector.database.SqliteDatabaseDriver.openDatabase(SqliteDatabaseDriver.java:211)
                                                                                             at com.facebook.stetho.inspector.database.SqliteDatabaseDriver.getTableNames(SqliteDatabaseDriver.java:121)
                                                                                             at com.facebook.stetho.inspector.protocol.module.Database.getDatabaseTableNames(Database.java:108)
                                                                                             at java.lang.reflect.Method.invoke(Native Method)
                                                                                             at com.facebook.stetho.inspector.MethodDispatcher$MethodDispatchHelper.invoke(MethodDispatcher.java:96)
                                                                                             at com.facebook.stetho.inspector.MethodDispatcher.dispatch(MethodDispatcher.java:67)
                                                                                             at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteRequest(ChromeDevtoolsServer.java:129)
                                                                                             at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteMessage(ChromeDevtoolsServer.java:111)
                                                                                             at com.facebook.stetho.inspector.ChromeDevtoolsServer.onMessage(ChromeDevtoolsServer.java:87)
                                                                                             at com.facebook.stetho.websocket.WebSocketSession$1.handleTextFrame(WebSocketSession.java:176)
                                                                                             at com.facebook.stetho.websocket.WebSocketSession$1.onCompleteFrame(WebSocketSession.java:136)
                                                                                             at com.facebook.stetho.websocket.ReadHandler.readLoop(ReadHandler.java:44)
                                                                                             at com.facebook.stetho.websocket.WebSocketSession.handle(WebSocketSession.java:45)
                                                                                             at com.facebook.stetho.websocket.WebSocketHandler.doUpgrade(WebSocketHandler.java:117)
                                                                                             at com.facebook.stetho.websocket.WebSocketHandler.handleRequest(WebSocketHandler.java:83)
                                                                                             at com.facebook.stetho.server.http.LightHttpServer.dispatchToHandler(LightHttpServer.java:84)
                                                                                             at com.facebook.stetho.server.http.LightHttpServer.serve(LightHttpServer.java:61)
                                                                                             at com.facebook.stetho.inspector.DevtoolsSocketHandler.onAccepted(DevtoolsSocketHandler.java:52)
                                                                                             at com.facebook.stetho.server.ProtocolDetectingSocketHandler.onSecured(ProtocolDetectingSocketHandler.java:63)
                                                                                             at com.facebook.stetho.server.SecureSocketHandler.onAccepted(SecureSocketHandler.java:33)
                                                                                             at com.facebook.stetho.server.LazySocketHandler.onAccepted(LazySocketHandler.java:36)
                                                                                             at com.facebook.stetho.server.LocalSocketServer$WorkerThread.run(LocalSocketServer.java:167)
09-13 13:55:51.768 7425-9609/com.just.another.app W/System.err: com.facebook.stetho.inspector.jsonrpc.JsonRpcException: INVALID_REQUEST: android.database.sqlite.SQLiteCantOpenDatabaseException: unknown error (code 14): Could not open database
09-13 13:55:51.768 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.protocol.module.Database.getDatabaseTableNames(Database.java:114)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.MethodDispatcher$MethodDispatchHelper.invoke(MethodDispatcher.java:96)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.MethodDispatcher.dispatch(MethodDispatcher.java:67)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteRequest(ChromeDevtoolsServer.java:129)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteMessage(ChromeDevtoolsServer.java:111)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.ChromeDevtoolsServer.onMessage(ChromeDevtoolsServer.java:87)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.WebSocketSession$1.handleTextFrame(WebSocketSession.java:176)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.WebSocketSession$1.onCompleteFrame(WebSocketSession.java:136)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.ReadHandler.readLoop(ReadHandler.java:44)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.WebSocketSession.handle(WebSocketSession.java:45)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.WebSocketHandler.doUpgrade(WebSocketHandler.java:117)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.websocket.WebSocketHandler.handleRequest(WebSocketHandler.java:83)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.http.LightHttpServer.dispatchToHandler(LightHttpServer.java:84)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.http.LightHttpServer.serve(LightHttpServer.java:61)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.inspector.DevtoolsSocketHandler.onAccepted(DevtoolsSocketHandler.java:52)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.ProtocolDetectingSocketHandler.onSecured(ProtocolDetectingSocketHandler.java:63)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.SecureSocketHandler.onAccepted(SecureSocketHandler.java:33)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.LazySocketHandler.onAccepted(LazySocketHandler.java:36)
09-13 13:55:51.769 7425-9609/com.just.another.app W/System.err:     at com.facebook.stetho.server.LocalSocketServer$WorkerThread.run(LocalSocketServer.java:167)
09-13 13:55:51.769 7425-9609/com.just.another.app W/ChromeDevtoolsServer: Error processing remote message
09-13 13:55:55.735 7425-9609/com.just.another.app D/ChromeDevtoolsServer: onClose: reason=1011 EOF while reading
09-13 13:55:55.736 7425-9609/com.just.another.app I/ResponseBodyFileManager: Cleaned up temporary network files.
```

**Comment by MohMah** [comment]
I have this issue, too. I think it it attempts to open `open(/actions.db)` instead of `open(actions.db)`

**Comment by longinoa** [comment]
See #431 this was fixed by #432 and a new version has just been released
